packages/dendron-11ty/raw-assets/sass/base.scss

```diff
@@ -92,6 +92,8 @@ pre {
 
 li {
   margin: 0.25em 0;
+  // Otherwise empty list items stack on top of each other
+  height: 1rem;
```

Comment:
(not a css expert)

Does this affect multiline contents? Should this be `min-height` instead?
```md
- something super ... long
- something with additional contents below
    > quote
- foo

Reply:
Good catch! That was a problem and using `min-height` fixes it.